docs/absolute.md

```diff
@@ -242,6 +242,10 @@ The `duration` argument can be any value that could be passed to `Temporal.Durat
 - any object with properties denoting a duration, such as `{ hours: 5, minutes: 30 }`;
 - a string in ISO 8601 duration format, such as `PT5H30M`.
 
+The `years` and `months` fields of `duration` must be zero, because adding a year or a month to a `Temporal.Absolute` is invalid and will throw a `RangeError`.
```

Comment:
Can you remind me where we discussed this behavior?

Reply:
It's been the case in the polyfill since before I started working on this project, so maybe @pipobscure knows?

In https://github.com/tc39/proposal-temporal/issues/307#issuecomment-588521748 and the following thread we discussed that Absolute.difference() never returns days or months, so in any case this makes sense to me accordingly.